src/blueapi/cli/rest.py

```diff
@@ -103,14 +104,27 @@ def _request_and_deserialize(
         data: Mapping[str, Any] | None = None,
         method="GET",
         raise_if: Callable[[requests.Response], bool] = _is_exception,
-    ) -> T:
+    ) -> T | BlueskyRemoteError :
```

Comment:
Unsure why the error is returned rather than raised

Reply:
I need to look into how to handle that from the CLI.

but tbh Python does not have an indicator on the function that it can throw some error. 

another solution is to always return a tuple [value, error], where each could be null

Reply:
Why not...?
```python
try:
    client.foo()
except BlueskyRemoteError:
    handle_error()
```

Reply:
@stan-dot presume you meant to comment this here:

> But the handle error must be in the cli. Otherwise you either quit the program or return something

You can handle the error in the `except` block

Reply:
ok done